### PR TITLE
doc: update doc on adding kill statement support

### PR DESCRIPTION
**Pull request by harshit-gangal**

### CODE REVIEW

File: content/en/docs/18.0/reference/compatibility/mysql-compatibility.md [hunk 1]
```diff
@@ -125,17 +125,23 @@ Vitess does not yet support Window Functions or Common Table Expressions.
 
 ### Killing running queries
 
-Vitess does not yet support killing running queries via the `KILL` command through VTGate.
-Currently, there exists a comment directive [QUERY_TIMEOUT_MS](../../../user-guides/configuration-advanced/comment-directives/#query-timeouts-query_timeout_ms) that can be set to reduce the query timeout which will ensure that the query either return result or abort within that time.
+Vitess added support for killing running queries via the `KILL` command through VTGate in v18.
+To use the kill query statement, a new connection is requires as the current connection will be busy executing the query itself.?
```
Comment (deepthi) [inline content/en/docs/18.0/reference/compatibility/mysql-compatibility.md]:
```suggestion
To use the kill query statement, a new connection is required as the current connection will be busy executing the query.
```

File: content/en/docs/18.0/reference/compatibility/mysql-compatibility.md [hunk 2]
```diff
@@ -125,17 +125,23 @@ Vitess does not yet support Window Functions or Common Table Expressions.
...
+Similarly, how mysql client does on `ctrl+c` on the shell.
```
Comment (deepthi) [inline content/en/docs/18.0/reference/compatibility/mysql-compatibility.md]: Can you rewrite this sentence? It is not grammatical.

File: content/en/docs/18.0/reference/compatibility/mysql-compatibility.md [hunk 3]
```diff
@@ -125,17 +125,23 @@ Vitess does not yet support Window Functions or Common Table Expressions.
...
 
-Vitess does have strict query timeouts for OLTP workloads (see below). If you need to kill a query, you can connect to the underlying MySQL shard instance and run `KILL` from there.
+Alternatively, 
+- [query_timeout_ms](../../../user-guides/configuration-advanced/comment-directives/#query-timeouts-query_timeout_ms) query comment directive can be set to have query timeout which will ensure that the query either return result or abort within that time.
+- [mysql_server_query_timeout](../../programs/vtgate/) command line flag can be set on VTGate to have a default timeout.
+
+Vitess have strict query timeouts for OLTP workloads (see below).
```
Comment (deepthi) [inline content/en/docs/18.0/reference/compatibility/mysql-compatibility.md]:
```suggestion
Vitess does have strict query timeouts for OLTP workloads (see below).
```

File: content/en/docs/18.0/reference/compatibility/mysql-compatibility.md [hunk 4]
```diff
@@ -125,17 +125,23 @@ Vitess does not yet support Window Functions or Common Table Expressions.
...
 
-Vitess does not yet support killing running queries via the `KILL` command through VTGate.
-Currently, there exists a comment directive [QUERY_TIMEOUT_MS](../../../user-guides/configuration-advanced/comment-directives/#query-timeouts-query_timeout_ms) that can be set to reduce the query timeout which will ensure that the query either return result or abort within that time.
+Vitess added support for killing running queries via the `KILL` command through VTGate in v18.
+To use the kill query statement, a new connection is requires as the current connection will be busy executing the query itself.?
+Similarly, how mysql client does on `ctrl+c` on the shell.
 
-Vitess does have strict query timeouts for OLTP workloads (see below). If you need to kill a query, you can connect to the underlying MySQL shard instance and run `KILL` from there.
+Alternatively, 
+- [query_timeout_ms](../../../user-guides/configuration-advanced/comment-directives/#query-timeouts-query_timeout_ms) query comment directive can be set to have query timeout which will ensure that the query either return result or abort within that time.
```
Comment (deepthi) [inline content/en/docs/18.0/reference/compatibility/mysql-compatibility.md]: Should be rewritten to be grammatical. I recommend ChatGPT as copy editor 😄

File: content/en/docs/18.0/reference/compatibility/mysql-compatibility.md [hunk 5]
```diff
@@ -125,17 +125,23 @@ Vitess does not yet support Window Functions or Common Table Expressions.
...
+
+Vitess have strict query timeouts for OLTP workloads (see below).
 
 ### Workload
 
 By default, Vitess sets some intentional restrictions on the execution time and number of rows that a query can return. This default workload mode is called `OLTP`. This can be disabled by setting the workload to `OLAP`:
 ```sql
 SET workload='olap'
 ```
+These timeout are present on [vttablet](../../programs/vttablet/) like `queryserver-config-query-timeout`, `queryserver-config-transaction-timeout`
```
Comment (deepthi) [inline content/en/docs/18.0/reference/compatibility/mysql-compatibility.md]: Not sure why this new sentence is added here. What are you trying to convey?
That these are the timeouts enforced by vttablet (as a follow up to the previous statement)? Or is the intent to tell people what the flags are to override the defaults?

File: content/en/docs/18.0/reference/compatibility/mysql-compatibility.md (hunk 2 above)
Comment (GuptaManan100) [reply content/en/docs/18.0/reference/compatibility/mysql-compatibility.md]: `Similar to how MySQL client behaves on...` or `Exactly like MySQL, when the client...`

File: content/en/docs/18.0/reference/compatibility/mysql-compatibility.md (hunk 5 above)
Comment (harshit-gangal) [reply content/en/docs/18.0/reference/compatibility/mysql-compatibility.md]: improved the docs with next commit.

**Review by frouioui — APPROVED** [review]

**Pull request merged**